fendermint/vm/topdown/src/sync.rs

```diff
@@ -195,6 +306,14 @@ impl IPCAgentProxy {
         Ok(height as BlockHeight)
     }
 
+    pub async fn get_genesis_epoch(&self, _subnet_id: &SubnetID) -> anyhow::Result<ChainEpoch> {
```

Comment:
Can you add some docstrings to explain the rationale of this method? 
* Why does it take `SubnetID` parameter if it doesn't use it?
* What is `self.child_subnet` of an agent?
* What is a `genesis_epoch` of a `ValidatorSet`?
* What does the `get_validator_set` return when it's called with `None`?

Reply:
I assumed everyone is aware... Doc string added.

Reply:
Can't see the docs :thinking: